objcgen/methodhelper.cs

```diff
@@ -86,9 +87,14 @@ public void WriteInvoke (string args)
 				}
 			}
 
+			var method = "__method";
+			if (IsVirtual) {
+				implementation.WriteLine ($"MonoMethod* __virtual_method = mono_object_get_virtual_method ({instance}, __method);");
```

Comment:
^ @tritao I did not see any call to this function in `master` so you likely have a similar issue with virtual calls.